### DISCUSSION TITLE
Command Line thinks i'm a different user

### DISCUSSION THREAD

**Discussion by geymin** [community/community#22837] [New to GitHub]
<p>I am trying to push a 10GB change which is apparently too big for Github to push at once so apparently there’s a command line for segmenting the push. I tried to use it but command line is behaving like i’m in a different account despite me being logged in on this account on both <a href="http://Github.com" rel="noopener nofollow ugc">Github.com</a> and Github desktop.<br>
How do I change what account i’m logged in on in command line?</p>
<p><img src="https://user-images.githubusercontent.com/46097696/181073330-689f3287-fc53-4b9c-9501-ff098fd5f66e.png" alt="image" data-base62-sha1="7rlDccHK9enWREwYwM0zgrRcW3U" width="403" height="15"></p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/geymin">@geymin</a>, welcome to the community! You can work with all repositories on GitHub over SSH, although firewalls and proxys might refuse to allow SSH connections. Using SSH requires you to generate an SSH public/private keypair on your local machine and add the public key to your GitHub account. Every time you use Git to authenticate with GitHub, you’ll be prompted to enter your SSH key passphrase, unless you’ve <a href="https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent">stored the key</a>. For more information, see "<a href="https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent">Generating a new SSH key and adding it to the ssh-agent</a>.</p>
<p>I hope this helps</p>

**Comment by geymin** [comment]
<p>Thank you, I didn’t have an SSH key on my current account but the problem i’m having is command line thinks i’m in an old account. Adding a key to my current account didn’t change what account Command Line thinks i’m logged in on.</p>

**Comment by geymin** [comment, marked as answer]
<p>Alright, I fixed it by deleting all my github credentials from the credentials manager and logging back in.</p>
<p>I don’t see why the 2GB chunk command can’t just automatically happen if the push is too big on github desktop?</p>
